isitsnowinginberlin.py

```diff
@@ -86,28 +67,19 @@ def get_weather(key='cached-wx', update_redis=True):
         return update_weather(update_redis)
 
 
-def get_forecast(key='cached-forecast', update_redis=True):
-    cached_str = redis_store.get(key)
-    if cached_str:
-        logger.info("Using cached forecast")
-        return json.loads(cached_str)
-    else:
-        logger.info("Need to fetch forecast")
-        return update_forecast(update_redis)
-
-
 def is_snowing(wx):
-    return any("Snow" in item['main'] for item in wx['weather'])
+    return wx['currently']['icon'] == 'snow'
 
 
-def will_snow(forecast):
+def will_snow(wx):
     now = datetime.now()
-    return any(is_snowing(element) for element in forecast['list'] if now < datetime.fromtimestamp(element['dt']))
+    next_forecast = next(data for data in wx['daily']['data'] if now < datetime.fromtimestamp(data['time']))
```

Comment:
Maybe need to check more than just one forecast reading (e.g. check for date range until tomorrow)